fetch.bs

```diff
@@ -460,12 +449,20 @@ each other by 0x2C 0x20, in order.
    <!-- Maybe give Infra "byte-alphanumeric"? -->
 
    <dt>`<code>content-type</code>`
-   <dd><p>If <var>value</var>, <a lt="extract header values">once extracted</a>, does not have a
-   MIME type (ignoring parameters) that is <code>application/x-www-form-urlencoded</code>,
-   <code>multipart/form-data</code>, or <code>text/plain</code>, then return false.
-   <!-- XXX * needs better xref
-            * ignoring parameters has been the standard for a long time now
-            * interesting test: "Content-Type: text/plain;" -->
+   <dd>
+    <ol>
+     <li><p>If <var>value</var> contains a <a>CORS-unsafe request-header byte</a>, then return
+     false.
+
+     <li><p>Let <var>mimeType</var> be the result of <a lt="parse a MIME type">parsing</a>
+     <var>value</var>.
+
+     <li><p>If <var>mimeType</var> is falure, then return false.
+
+     <li><p>If <var>mimeType</var>'s <a for="MIME type">essence</a> is not
+     "<code>application/x-www-form-urlencoded</code>", "<code>multipart/form-data</code>", or
+     "<code>text/plain</code>", then return false.
+    </ol>
```

Comment:
It would be good to add a note here illustrating that this is very different from how response Content-Type is parsed, but maybe that should wait until that's actually defined properly.

Reply:
I'll do this as part of #529.